client/components/editor/teaching-elements/Assessment/ImageMultipleChoice.vue

```diff
@@ -106,45 +115,36 @@ export default {
       return new Promise((resolve, reject) => {
         this.resetErrorMessages(dataset.index);
         if (imageFile.size > 100000) {
-          // eslint-disable-next-line prefer-promise-reject-errors
-          return reject('The image is too large. Maximum allowed image size: 100 kB');
+          return reject(new Error('The image is too large. Maximum allowed image size: 100 kB'));
         }
         return resolve(imageFile);
       })
         .then(() => blobToBase64String(imageFile))
```

Comment:
Or you could use: [blobToDataURL](https://github.com/nolanlawson/blob-util#blobtodataurl) and avoid implementing and calling `getImageDataUrl` twice...